docs/index.md

```diff
@@ -8,22 +8,28 @@
 :end-before: (DO-NOT-REMOVE-docs-intro-END)
 ```
 
-
 ```{toctree}
 :maxdepth: 2
-:caption: Usage
+:caption: Installation
 
-subcommands
-faq
+installation
 ```
 
 ```{toctree}
 :maxdepth: 2
-:caption: Configuring
+:caption: Configuration
```

Comment:
You're refactoring the ToC titles in your instructions PR. Could you please move this effort to a separate one? Plus there's `Contributing` that is inconsistent with this change.